google/cloud/bigtable/data/_async/client.py

```diff
@@ -486,13 +487,15 @@ async def _register_instance(
 
         Args:
           instance_id: id of the instance to register.
-          owner: table that owns the instance. Owners will be tracked in
+          app_profile_id: id of the app profile calling the instance.
+          owner_id: integer id of the object owning the instance. Owners will be tracked in
```

Comment:
nit: maybe reference that this uses the id() identity function?

Doesn't matter too much since this is internal, but it might not be clear what id means in this context. Especially since we have table_id, app_profile_id, instance_id, etc

Reply:
Updated